packages/shared/src/element.ts

```diff
@@ -1,176 +1,18 @@
-const HTMLTagSet = new Set([
-  'html',
-  'body',
-  'base',
-  'head',
-  'link',
-  'meta',
-  'style',
-  'title',
-  'address',
-  'article',
-  'aside',
-  'footer',
-  'header',
-  'h1',
-  'h2',
-  'h3',
-  'h4',
-  'h5',
-  'h6',
-  'hgroup',
-  'nav',
-  'section',
-  'div',
-  'dd',
-  'dl',
-  'dt',
-  'figcaption',
-  'figure',
-  'picture',
-  'hr',
-  'img',
-  'li',
-  'main',
-  'ol',
-  'p',
-  'pre',
-  'ul',
-  'a',
-  'b',
-  'abbr',
-  'bdi',
-  'bdo',
-  'br',
-  'cite',
-  'code',
-  'data',
-  'dfn',
-  'em',
-  'i',
-  'kbd',
-  'mark',
-  'q',
-  'rp',
-  'rt',
-  'rtc',
-  'ruby',
-  's',
-  'samp',
-  'small',
-  'span',
-  'strong',
-  'sub',
-  'sup',
-  'time',
-  'u',
-  'var',
-  'wbr',
-  'area',
-  'audio',
-  'map',
-  'track',
-  'video',
-  'embed',
-  'object',
-  'param',
-  'source',
-  'canvas',
-  'script',
-  'noscript',
-  'del',
-  'ins',
-  'caption',
-  'col',
-  'colgroup',
-  'table',
-  'thead',
-  'tbody',
-  'td',
-  'th',
-  'tr',
-  'button',
-  'datalist',
-  'fieldset',
-  'form',
-  'input',
-  'label',
-  'legend',
-  'meter',
-  'optgroup',
-  'option',
-  'output',
-  'progress',
-  'select',
-  'textarea',
-  'details',
-  'dialog',
-  'menu',
-  'menuitem',
-  'summary',
-  'content',
-  'element',
-  'shadow',
-  'template',
-  'blockquote',
-  'iframe',
-  'tfoot'
-])
+import { makeMap } from './makeMap'
+
+const HTML_TAGS =
+  'html,body,base,head,link,meta,style,title,address,article,aside,footer,header,h1,h2,h3,h4,h5,h6,hgroup,nav,section,div,dd,dl,dt,figcaption,figure,picture,hr,img,li,main,ol,p,pre,ul,a,b,abbr,bdi,bdo,br,cite,code,data,dfn,em,i,kbd,mark,q,rp,rt,rtc,ruby,s,samp,small,span,strong,sub,sup,time,u,var,wbr,area,audio,map,track,video,embed,object,param,source,canvas,script,noscript,del,ins,caption,col,colgroup,table,thead,tbody,td,th,tr,button,datalist,fieldset,form,input,label,legend,meter,optgroup,option,output,progress,select,textarea,details,dialog,menu,menuitem,summary,content,element,shadow,template,blockquote,iframe,tfoot'
```

Comment:
Seems that your linter doesn't work well, maximum line length is 80.

https://github.com/vuejs/vue-next/blob/1fbb4d22fa168a44a76e31d3fefcf633d5c4c4bf/.prettierrc#L3